src/web/client/common/remoteFetchProvider.ts

```diff
@@ -93,13 +94,13 @@ function createContentFiles(result: string, entity: string, queryParamsMap: any,
         vscode.window.showInformationMessage(patchRequestUrl)
         await saveData(accessToken, patchRequestUrl, e.uri, entity, saveDataMap, new TextDecoder(CHARSET).decode(newFileData));
     });
-    return saveDataMap;
 }
 
-function createVirtualFile(portalsFS: PortalsFS, fileName: string, languageCode: string, data: any, portalFileExtension: string, subUri: string) {
+function createVirtualFile(portalsFS: PortalsFS, fileName: string, languageCode: string, data: any, portalFileExtension: string, subUri: string,entityId : string, saveDataAtribute : string ) {
+    const saveEntityDetails = new SaveEntityDetails(entityId, entityName, saveDataAtribute);
```

Comment:
from where are you fetching the entityName? I don't see it being passed in the createVirtualFile function.